modules/simulator.js

```diff
@@ -72,8 +77,36 @@ function createOrganism(land)
     var randomMass = Constant.MIN_MASS + (Math.random()*Math.random()) * Constant.MAX_MASS;
     var randomMinSpeed =  (Math.random()) + Constant.MIN_SPEED;
 
-    // create Organism(generation, mass, energy, x, y, vision_angle, vision_range, minSpeed, fertility)
-    return new Organism(1, randomMass, randomX, randomY, 100, 100, randomMinSpeed, 1);
+    // 
+    if(organismCounter%3 == 0) {
```

Comment:
I'd suggest creating a "Type" enum class instead of hard-coded values (0,1,2)

ex. so that you can do something like
organismCounter == Type.MASS or Type.SPEED or Type.VISION